# AdvCiv-SAS reachable-history commit diff
# Email privacy: email-shaped addresses redacted from generated commit-history text.
# History metadata layout: 3 (history-segment label; title-only diff header; full message redirected to anonymized Git logs).
# Cache format: 2
# Cache policy: 8c87d6e4bd0f
# Practical commit count: 2875
# Commit: 0fa67425aef40854d5269e7926b0eeb912b4b91c
# History segment: pre-SAS AdvCiv history
# Parent used for diff: 644b94afe2b5b7bcb1415cf8e172c00cd89d74cc
# Title: Say where GPL license is found in Civ4lerts comments
# Full commit message/metadata: see _0_Common_Docs/git_logs/git_log_anonymized_email_002_Base_AdvCiv.txt.
# History segments describe ancestry/log partition, not authorship: SASBranch includes SAS work plus later upstream AdvCiv commits merged/imported into this branch.
# Practical commit counts can repeat on divergent/merged history; the full Git SHA is the canonical unique commit identifier.
# Diff policy: preserve textual source/docs/config history; summarize known generated/binary/noisy or exceptionally huge file patches.
# Git textconv/external diff drivers are disabled; redundant document formats such as ODT/PDF are summarized rather than converted.

[CHANGE SUMMARY - NUMSTAT + RENAME/MODE SUMMARY]
1	1	Assets/Python/Contrib/Civ4lerts.py
1	0	Assets/XML/Text/Civ4lerts_CIV4GameText.xml

[PATCHES]

diff --git a/Assets/Python/Contrib/Civ4lerts.py b/Assets/Python/Contrib/Civ4lerts.py
index 8ca5ef765..21769018d 100644
--- a/Assets/Python/Contrib/Civ4lerts.py
+++ b/Assets/Python/Contrib/Civ4lerts.py
@@ -16,9 +16,9 @@
 ## along with Civilization IV Alerts mod; if not, write to the Free
 ## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
 ## 02110-1301 USA
+## advc (note): The license is in the BUG Doc folder.
 
 __version__ = "$Revision: 1.2 $"
-# $Source: /usr/local/cvsroot/Civ4lerts/src/main/python/Civ4lerts.py,v $
 
 ## Civ4lerts
 ## This class extends the built in event manager and overrides various

diff --git a/Assets/XML/Text/Civ4lerts_CIV4GameText.xml b/Assets/XML/Text/Civ4lerts_CIV4GameText.xml
index 1c64973d0..d8049a755 100644
--- a/Assets/XML/Text/Civ4lerts_CIV4GameText.xml
+++ b/Assets/XML/Text/Civ4lerts_CIV4GameText.xml
@@ -18,6 +18,7 @@
   along with Civilization IV Alerts mod; if not, write to the Free
   Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
   02110-1301 USA
+  advc (note): The license is in the BUG Doc folder.
   -->

[PATCH COVERAGE]
patchFiles=2 embeddedFiles=2 omittedFiles=0 embeddedPatchBytes=1097
